test/tests/functional/pbs_hook_timeout.py

```diff
@@ -155,12 +162,16 @@ def test_hook_send(self):
 
         # Ensure that hook send updates are retried for
         # the *.HK and *.PY file to momB
-        self.server.log_match(
-            "Timing out previous send of mom hook updates "
-            "(send replies expected=2 received=0)", n=600,
-            max_attempts=timeout_max_attempt, interval=30,
-            starttime=start_time)
-
+        for count in range(10):
```

Comment:
code duplication: put this in a common method and invoke it from the 2 places instead. You can pass a "num_instances" arg which will be set to '2' for the first call and '3' for the second.